src/com/google/javascript/jscomp/ProcessCommonJSModules.java

```diff
@@ -1061,7 +1061,11 @@ private void updateNameReference(
               Node assign = IR.assign(getProp, nameRef.getFirstChild().detachFromParent());
               assign.setJSDocInfo(info);
               Node expr = IR.exprResult(assign).useSourceInfoIfMissingFromForTree(nameRef);
-              parent.replaceWith(expr);
```

Comment:
Thanks for the review. Is there an example somewhere in the code where in the a statement is split up?